static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.ts

```diff
@@ -468,27 +468,27 @@ const biddersSwitchedOn = (allBidders: PrebidBidder[]): PrebidBidder[] => {
 };
 
 const currentBidders = (slotSizes: HeaderBiddingSize[]): PrebidBidder[] => {
-	const otherBidders: PrebidBidder[] = [
-		...(inPbTestOr(shouldIncludeCriteo()) ? [criteoBidder] : []),
-		...(inPbTestOr(shouldIncludeSmart()) ? [smartBidder] : []),
-		...(inPbTestOr(shouldIncludeSonobi()) ? [sonobiBidder] : []),
-		...(inPbTestOr(shouldIncludeTrustX()) ? [trustXBidder] : []),
-		...(inPbTestOr(shouldIncludeTripleLift()) ? [tripleLiftBidder] : []),
-		...(inPbTestOr(shouldIncludeAppNexus()) ? [appNexusBidder] : []),
-		...(inPbTestOr(shouldIncludeImproveDigital())
-			? [improveDigitalBidder]
-			: []),
-		...(inPbTestOr(shouldIncludeImproveDigitalSkin())
-			? [improveDigitalSkinBidder]
-			: []),
-		...(inPbTestOr(shouldIncludeXaxis()) ? [xaxisBidder] : []),
-		pubmaticBidder,
-		...(inPbTestOr(shouldIncludeAdYouLike(slotSizes))
-			? [adYouLikeBidder]
-			: []),
-		...(inPbTestOr(shouldUseOzoneAdaptor()) ? [ozoneClientSideBidder] : []),
-		...(shouldIncludeOpenx() ? [openxClientSideBidder] : []),
+	const biddersToCheck: Array<[boolean, PrebidBidder]> = [
+		[shouldIncludeCriteo(), criteoBidder],
+		[shouldIncludeSmart(), smartBidder],
+		[shouldIncludeSonobi(), sonobiBidder],
+		[shouldIncludeTrustX(), trustXBidder],
+		[shouldIncludeTripleLift(), tripleLiftBidder],
+		[shouldIncludeAppNexus(), appNexusBidder],
+		[shouldIncludeImproveDigital(), improveDigitalBidder],
+		[shouldIncludeImproveDigitalSkin(), improveDigitalSkinBidder],
+		[shouldIncludeXaxis(), xaxisBidder],
+		[true, pubmaticBidder],
+		[shouldIncludeAdYouLike(slotSizes), adYouLikeBidder],
+		[shouldUseOzoneAdaptor(), ozoneClientSideBidder],
+		[shouldIncludeOpenx(), openxClientSideBidder],
 	];
+
+	const otherBidders: PrebidBidder[] = biddersToCheck.flatMap(
```

Comment:
Nice!

Sorry to ask this but do we polyfill flatMap for IE11?

Do we even care about IE11?

Reply:
Argh I forgot about IE11! I guess erring on the side of caution, and given this is a simple usage of flatMap anyway, I can replace with a filter->map.